index.html

```diff
@@ -0,0 +1,132 @@
+                    </ul>
+                </div>
+            </nav>
+        </header>
+        <main>
+            <section id="homepage">
+                <div class="home-text">
+                    <img src="images/picture.jpg" alt="personal picture">
+                    <p class="name">Hi! I'm <span>Arinze I.C</span></p>
+                    <p class="desc">Front-end web developer</p>
+                    <a href="#">My Work</a>
+                </div>
+            </section>
+    </div>
+    <section id="social-media">
+        <h3>Let's connect:</h3>
+        <div>
+            <a href="https://www.instagram.com/ari.n.ze/" target="_blank"><i class="fab fa-instagram"></i></a>
+            <a href="https://www.linkedin.com/in/arinze-ihemedu-14b738148/" target="_blank"><i
+                    class="fab fa-linkedin-in"></i></a>
+            <a href="https://www.quora.com/profile/Arinze-Ihemedu" target="_blank"><i class="fab fa-quora"></i></a>
+            <a href="https://github.com/Arinzemike1" target="_blank"><i class="fab fa-github"></i></a>
+        </div>
+    </section>
+    <section id="about">
+        <div class="container">
+            <h1>About Me</h1>
+            <div class="row">
+                <div class="col-md-6 text-center about-col">
+                    <div class="profile">
+                        <img src="images/picpic.jpg" class="user" alt="about image">
+                        <p>Hi there! I'm Arinze Ihemedu, a graduate of applied informatics and system programming from
+                            Vitebsk State University. I've got some basic understanding of c++ programming language but
+                            here I am doing what I love most which is web development. My stack: HTML5, CSS3/Bootstrap,
+                            Javascript/Jquery.</p>
+                    </div>
+                </div>
+                <div class="col-md-6 skills-bar">
+                    <p>HTML</p>
+                    <div class="progress">
+                        <div class="progress-bar" style="width: 85%;">80%</div>
+                    </div>
+                    <p>CSS</p>
+                    <div class="progress">
+                        <div class="progress-bar" style="width: 70%;">60%</div>
+                    </div>
+                    <p>Javascript</p>
+                    <div class="progress">
+                        <div class="progress-bar" style="width: 40%;">40%</div>
+                    </div>
+                    <p>Wordpress</p>
+                    <div class="progress">
+                        <div class="progress-bar" style="width: 70%;">70%</div>
+                    </div>
+                    <p>GIT</p>
+                    <div class="progress">
+                        <div class="progress-bar" style="width: 30%;">50%</div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="contact-btn"><a href="#">Contact me</a></div>
```

Comment:
This section should contain a form that collects user's message eventually and sends to your mail. For now it should just be a form